engine.py

```diff
@@ -123,6 +123,7 @@ def __setup_ui(self, callback):
         t = tk_shotgun.Task(self, callback)
         
         # start up our QApp now
+        QtGui.QApplication.setLibraryPaths([])
```

Comment:
This happens when I try to start flame from the browser integration.
![action1](https://cloud.githubusercontent.com/assets/6785406/26464964/8de2f1da-4157-11e7-8a9d-641ab29353b8.png)